docs/en/01-how-to-build/android.md

```diff
@@ -198,3 +129,22 @@ MMDeploy provides a recipe as shown below for building SDK with ncnn as inferenc
 
   make -j$(nproc) && make install
   ```
+
+Please check [cmake build option spec](cmake_option.md)
+
+#### Build Demo
+
+```Bash
+export ANDROID_ABI=arm64-v8a
+
+cd ${MMDEPLOY_DIR}/build_${ANDROID_ABI}/install/example
```

Comment:
maybe is `cd ${MMDEPLOY_DIR}/build_${ANDROID_ABI}/install/example/cpp`
The same as zh-cn version, and the demo build for other unchanged platform.

Reply:
done.